app/controllers/public_recipes_controller.rb

```diff
@@ -1,5 +1,6 @@
 class PublicRecipesController < ApplicationController
   def index
     @recipes = Recipe.where(public: true)
+    @recipes = Recipe.includes(%i[user recipe_foods])
   end
```

Comment:
- [ ] Kindly ensure your public recipe wireframe matches the one given:


Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/202848876-f385f010-4025-45ca-bba6-1ad9f1e631fe.png)|  ![image](https://user-images.githubusercontent.com/61408860/202848904-600946b6-8c1e-4b2c-b776-764167308386.png)

Reply:
done